Kbs.Wpf/Boat/Details/BoatDetailPage.xaml.cs

```diff
@@ -20,16 +22,36 @@ public partial class BoatDetailPage : Page
     private readonly UserRepository _userRepository = new();
     private readonly ReservationRepository _registrationRepository = new();
     private readonly INavigationManager _navigationManager;
+    private const int _daysForDeletion = 30;
```

Comment:
Zou ik in de businesss laag zetten. Kan bijvoorbeeld in de BoatValidator. Dan maak je daar een nieuwe methode in aan: `IsValidForPermanentDeletion`